storage.go

```diff
@@ -216,28 +216,8 @@ func (s *Storage) Boot(ctx context.Context) error {
 	return nil
 }
 
-func (s *Storage) Create(ctx context.Context, key, value string) error {
-	var err error
-
-	key, err = microstorage.SanitizeKey(key)
-	if err != nil {
-		return microerror.Mask(err)
-	}
-
-	err = s.Put(ctx, key, value)
-	if err != nil {
-		microerror.Mask(err)
-	}
-	return nil
-}
-
-func (s *Storage) Put(ctx context.Context, key, value string) error {
-	var err error
-
-	key, err = microstorage.SanitizeKey(key)
-	if err != nil {
-		return microerror.Mask(err)
-	}
+func (s *Storage) Put(ctx context.Context, kv microstorage.KV) error {
+	key, value := kv.Key(), kv.Val()
```

Comment:
We don't need sanitization anymore. KV and K are immutable and validated upon creation. This is also reflected in the test suite.

Reply:
I really like this 👍